_posts/2018-11-16-grpc-dotnet-build.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: gRPC Meets .NET SDK And Visual Studio
+published: true
+permalink: blog/grpc-dotnet-build
+author: Kirill 'kkm' Katsnelson
+author-link: https://github.com/kkm000
+company: SmartAction
+company-link: https://www.smartaction.ai
+---
+As part of Microsoft's move towards its cross-platform .NET offering, they have
```

Comment:
Add a new line between the `---`, since the GitHub MD parser is wonky.

Reply:
Done. I have no idea why does not it show up in the diff on the convo page. Apparently, it's not only the MD parser that is wonky on GitHub...

Btw, there are other .md blog entries without a newline. Do you want me to add newlines to all, just in case? Can do a PR for that.

Reply:
That's alright.  This post is already a lot of work.  (But if you want to sure!)

Reply:
No biggie. Opened #779. There were only 3 files w/o an empty line under _posts/.